.travis.yml

```diff
@@ -29,7 +31,15 @@ deploy:
     script: bash scripts/release
     on:
       branch: master
+      condition: $DEPLOY = true
```

Comment:
```suggestion
```

This part will not work here as we don't have any "DEPLOY=true" on test matrix, just one job.